tools/azure/cleanup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nash
+
+resgroups <= azure group list
+resgroups <= split($resgroups, "\n")
+
+for resgroup in $resgroups {
+    filtered <= echo $resgroup | -grep "klb"
+    echo "filtered["+$filtered+"]"
+    splitted <= split($filtered, " ")
+    size <= len($splitted)
+    echo "size: " + $size
+    for word in $splitted {
+        echo "word[" + $word + "]"
+    }
```

Comment:
@tiago4orion I tried a different approach and stumbled on a interesting behavior of split, if I split by spaces, it will split by individual spaces, and produce a lot of empty strings. Example of output:

```
filtered[]
size: 1
word[]
filtered[]
size: 1
word[]
filtered[]
size: 1
word[]
filtered[]
size: 1
word[]
filtered[data:    klb-resgroup-tests-763533  eastus    Succeeded           null ]
size: 23
word[data:]
word[]
word[]
word[]
word[klb-resgroup-tests-763533]
word[]
word[eastus]
word[]
word[]
word[]
word[Succeeded]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[null]
word[]
filtered[data:    klb-resgroup-tests-968791  eastus    Succeeded           null ]
size: 23
word[data:]
word[]
word[]
word[]
word[klb-resgroup-tests-968791]
word[]
word[eastus]
word[]
word[]
word[]
word[Succeeded]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[]
word[null]
word[]
filtered[]
size: 1
word[]
```

It does not seem like a cool behavior for a split function, what you think ?